README.md

```diff
@@ -13,6 +13,18 @@ _Please note that although the below examples are written in interactive style,
 actually work in the interactive interpreter._
 
 
+####API changes in 1.2
+From version 1.2, the behavior of parallel filter method was changed to same with CPython's built-in filter.
+The differences with previous version are as follows:
+
+* It became able to use `None` as a predicate for filter.
+  * In previous versions, it raises.
+
```

Comment:
It's very nice. I'll change it to:

* It is now possible to use ``None`` as a predicate for ``filter``
  * Previously, it would raise a ``TypeError``.